api/communication-api/src/test/java/com/cognifide/aet/communication/api/metadata/SerializingOperationTest.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Automated Exploratory Tests
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.aet.communication.api.metadata;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.google.gson.Gson;
+
+public class SerializingOperationTest {
+
+  @Test
+  public void testSerialization() throws Exception {
+    Operation operationInput = new Operation("test");
+    operationInput.addParameter("key-test", "value-test");
+    Assert.assertFalse(operationInput.getParameters().isEmpty());
+    Assert.assertEquals(1, operationInput.getParameters().size());
+
+    Gson gson = new Gson();
+    String json = gson.toJson(operationInput);
+    Operation operationOutput = gson.fromJson(json, Operation.class);

```

Comment:
Please create `Operation` instance from te json file that is e.g. read from the resources, not created before from `Operation` instance.